### DISCUSSION TITLE
Allow the ability to download Autoplay plays

### DISCUSSION THREAD

**Discussion by localcake** [ppy/osu#14243] [Ideas] [answered]
Whenever a play by Autoplay is done there doesn't seem to be the ability to download it.
![image](https://user-images.githubusercontent.com/88810043/129127112-76ef35db-be19-4a24-8321-7ea7fbbf14cc.png)

This also ties into a bug in which the accuracy of replays can be tampered with if the player scrolls through a replay rapidly (as seen in the 98% accuracy by Autoplay).

**Comment by peppy** [comment, marked as answer]
why would you need this?

**Reply by localcake** [reply]
I like to download Autoplay replays for fun and then process them through [this website](https://ordr.issou.best/#/) to make them look nice...

**Reply by peppy** [reply]
That's a bit.. out of scope. I'm not sure we will/can support this going forward.

**Reply by localcake** [reply]
That's okay thank you

**Reply by PercyDan54** [reply]
> That's okay thank you

Out of scope in this repo but I just want to say here...
You can just download [danser](https://github.com/Wieku/danser-go) and set mover to `linear` to get autoplay behaviour.
